### PR TITLE
Use single quotes consistently, everywhere.

### PR DESCRIPTION
**Pull request by lewiscollard**
Following on from #69. Use single quotes everywhere to replace the mixture of double and single quotes.

### CODE REVIEW

File: {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/urls.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-"""URL config for {{cookiecutter.repo_name}} project."""
+'''URL config for {{cookiecutter.repo_name}} project.'''
```
Comment (danielsamuels) [inline {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/urls.py]: Can probably drop this

Comment (lewiscollard) [reply {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/urls.py]: I was thinking the same.

Can the same be done for `wsgi.py` too? That and the NewRelic archaeological find..

Comment (danielsamuels) [reply {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/urls.py]: Sure

Comment (lewiscollard) [reply {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/urls.py]: Done :)

**Pull request merged**